deploy-board/deploy_board/webapp/helpers/accounts_helper.py

```diff
@@ -15,11 +15,14 @@ def get_all_accounts(request):
         return None
 
 
-def get_by_cell_and_id(request, cell, account_id):
+def get_by_cell_and_id(request, cell, account_id, provider="AWS"):
     try:
-        return rodimus_client.get(f"/accounts/AWS/{cell}/{account_id}", request.teletraan_user_id.token)
+        return rodimus_client.get(f"/accounts/{provider}/{cell}/{account_id}", request.teletraan_user_id.token)
     except Exception as e:
         log.error(f"Can't get account by cell and account_id: "
                   f"cell = {cell}, account_id = {account_id}, error = {e}")
         # return None for backward-compatibility
         return None
+    
+def get_default_account(request, cell, provider='AWS'):
+    return get_by_cell_and_id(request, cell, "default", provider)
```

Comment:
Let's add `try except` for backward compatibility: deploy board may be deployed before cluster management service, that's why `get_default_account` will return 404 error.

So please change this function to:
```Python
def get_default_account(request, cell, provider="AWS"):
    try:
        return get_by_cell_and_id(request, cell, "default", provider)
    except Exception as e:
        # log exception
        return None
```

Reply:
marking as resolved since exceptions should be caught by `get_by_cell_and_id`